optimization/optimizer.py

```diff
@@ -36,48 +37,60 @@ def load_course_map(course_name="COTA"):
                 clean_data.append(row)
         # Create the elevation profile
         elev_profile = []
+        stops = []
         for i in range(len(clean_data) - 1):
-            pitch = numpy.arctan(float(clean_data[i][1]) / 10)
-            elev_profile.append(pitch)
+            pitch = numpy.arctan(float(clean_data[i][1]) / 10)  # is 10 = 30/3? i.e. rise over run?
```

Comment:
Yep. Should clean that up

Reply:
Cleaned up in latest commit by adding ydir variability